modules/users/client/components/admin-user-form.component.ts

```diff
@@ -65,6 +74,38 @@ export class AdminUserForm {
     });
   }
 
+  deleteUser()
+  {
+    let confirmed: any = confirm("Are you sure you want to delete this user? This can't be undone.");
+    if(confirmed)
+    {
+      this.userService.delete(this.user._id).subscribe( result =>
+      {
+        if(result.success)
+        {
+           //notify success and tell parent to pull list of users from the server
+            this.notificationsService.info('User Deleted', 'The user has been deleted', {
+            timeOut: 3000,
```

Comment:
some common notificationsService config stuff can probably be factored out and reused for the info and error notifications